kornia/enhance/adjust.py

```diff
@@ -633,69 +633,116 @@ def _blend_one(input1: torch.Tensor, input2: torch.Tensor, factor: torch.Tensor)
     return torch.clamp(res, 0, 1)
 
 
-def _build_lut(histo, step):
-    # Compute the cumulative sum, shifting by step // 2
-    # and then normalization by step.
-    lut = (torch.cumsum(histo, 0) + (step // 2)) // step
-    # Shift lut, prepending with 0.
-    lut = torch.cat([torch.zeros(1, device=lut.device, dtype=lut.dtype), lut[:-1]])
-    # Clip the counts to be in range.  This is done
-    # in the C code for image.point.
-    return torch.clamp(lut, 0, 255)
```

Comment:
would be nice if you keep this functionality separated. Ideally we might want to have exposed to the API similar to this `cv::LUT` https://docs.opencv.org/master/d2/de8/group__core__array.html#gab55b8d062b7f5587720ede032d34156f

Reply:
sure, will do